Library/Homebrew/utils/popen.rb

```diff
@@ -14,7 +14,12 @@ def self.popen(args, mode, options = {})
         yield pipe
       else
         options[:err] ||= :close unless ENV["HOMEBREW_STDERR"]
-        exec(*args, options)
+        begin
+          exec(*args, options)
+        rescue SystemCallError
+          $stderr.puts "brew: command not found: #{args[0]}" unless options[:err] == :close
```

Comment:
This message doesn't seem to match the exception; presumably there are other `SystemCallError`s?

Reply:
> You should be able to catch Errno::ENOENT and inspect it if that's the problem.

Thanks for the tip. Fixed.